backend/hostgw/network.go

```diff
@@ -162,21 +177,21 @@ func (n *network) checkSubnetExistInRoutes() {
 				}
 			}
 			if !exist {
-				if err := netlink.RouteAdd(&route); err != nil {
+				if err := routes.AddRoute(&route); err != nil {
 					if nerr, ok := err.(net.Error); !ok {
-						log.Errorf("Error recovering route to %v: %v, %v", route.Dst, route.Gw, nerr)
+						log.Errorf("Error recovering route to %s: %s, %v", route.Destination.IP, route.Gateway, nerr)
 					}
 					continue
 				} else {
-					log.Infof("Route recovered %v : %v", route.Dst, route.Gw)
+					log.Infof("Route recovered %s : %s", route.Destination.IP, route.Gateway)
 				}
 			}
 		}
 	}
 }
 
-func routeEqual(x, y netlink.Route) bool {
-	if x.Dst.IP.Equal(y.Dst.IP) && x.Gw.Equal(y.Gw) && bytes.Equal(x.Dst.Mask, y.Dst.Mask) {
+func routeEqual(x, y routes.Route) bool {
+	if x.Destination.IP.Equal(y.Destination.IP) && x.Gateway.Equal(y.Gateway) && bytes.Equal(x.Destination.Mask, y.Destination.Mask) {
```

Comment:
This isn't a rename as much as a creating of a Route object to abstract between netlink and iphlpapi. I didn't want to create a copy/paste of the entire network.go file, so I created a small wrapper around the routes.